docs/graph_summary.json

```diff
@@ -0,0 +1 @@
_id.4629d76478", "type": "test"}, "32": {"name": "test.twilio_source.not_null_stg_twilio__address_address_id.88b6a635a6", "type": "test"}, "33": {"name": "test.twilio_source.unique_stg_twilio__call_call_id.4a911440b6", "type": "test"}, "34": {"name": "test.twilio_source.not_null_stg_twilio__call_call_id.b1c7afdc81", "type": "test"}, "35": {"name": "test.twilio_source.unique_stg_twilio__incoming_phone_number_incoming_phone_number_id.d6eb15efc6", "type": "test"}, "36": {"name": "test.twilio_source.not_null_stg_twilio__incoming_phone_number_incoming_phone_number_id.9a63d01724", "type": "test"}, "37": {"name": "test.twilio_source.not_null_stg_twilio__message_message_id.bc1b8551d4", "type": "test"}, "38": {"name": "test.twilio_source.not_null_stg_twilio__message_account_id.4e63c023d2", "type": "test"}, "39": {"name": "test.twilio_source.dbt_utils_unique_combination_of_columns_stg_twilio__message_account_id__message_id__messaging_service_id.2696e90a8e", "type": "test"}, "40": {"name": "test.twilio_source.not_null_stg_twilio__messaging_service_messaging_service_id.c7032fe946", "type": "test"}, "41": {"name": "test.twilio_source.not_null_stg_twilio__messaging_service_account_id.f46a18a677", "type": "test"}, "42": {"name": "test.twilio_source.dbt_utils_unique_combination_of_columns_stg_twilio__messaging_service_account_id__messaging_service_id.2de8528f3d", "type": "test"}, "43": {"name": "test.twilio_source.unique_stg_twilio__outgoing_caller_id_outgoing_caller_id.0e55c1c2ea", "type": "test"}, "44": {"name": "test.twilio_source.not_null_stg_twilio__outgoing_caller_id_outgoing_caller_id.de6e667878", "type": "test"}, "45": {"name": "test.twilio_source.not_null_stg_twilio__usage_record_account_id.8b387072fd", "type": "test"}, "46": {"name": "test.twilio_source.not_null_stg_twilio__usage_record_category.e6d57544df", "type": "test"}, "47": {"name": "test.twilio_source.not_null_stg_twilio__usage_record_end_date.0f8fc2406e", "type": "test"}, "48": {"name": "test.twilio_source.not_null_stg_twilio__usage_record_start_date.a73353be1b", "type": "test"}, "49": {"name": "test.twilio_source.dbt_utils_unique_combination_of_columns_stg_twilio__usage_record_category__start_date__end_date__account_id.3782e14f70", "type": "test"}, "50": {"name": "test.twilio.unique_twilio__number_overview_phone_number.3edf30e4ae", "type": "test"}, "51": {"name": "test.twilio.not_null_twilio__number_overview_phone_number.baa4731adf", "type": "test"}, "52": {"name": "test.twilio.unique_twilio__message_enhanced_message_id.715528efa4", "type": "test"}, "53": {"name": "test.twilio.not_null_twilio__message_enhanced_message_id.970ba0a457", "type": "test"}, "54": {"name": "test.twilio.not_null_twilio__account_overview_account_id.8981fee2f5", "type": "test"}, "55": {"name": "test.twilio.not_null_twilio__account_overview_date_day.fe10c23e08", "type": "test"}, "56": {"name": "test.twilio.dbt_utils_unique_combination_of_columns_twilio__account_overview_account_id__date_day.f43ddc7113", "type": "test"}}}
```

Comment:
@fivetran-reneeli I don't think we bring in the `graph_summary.json`. Can you remove this file from the docs folder.

Reply:
whoops-- fixed